tika-parsers/src/main/java/org/apache/tika/parser/csv/TextAndCSVParser.java

```diff
@@ -306,7 +306,6 @@ private CSVParams getOverride(Metadata metadata) {
 
         String delimiterString = mediaType.getParameters().get(DELIMITER);
         if (delimiterString == null) {
-            return new CSVParams(mediaType, charset);
```

Comment:
Why remove this?

Reply:
This removal was unintentional. Apologies for not double checking the diff.
Shall I create a new merge request?

Reply:
You can revert it and commit the changes to your branch. It'd automatically show up in this PR.